source/docs/hardware/hardware-basics/status-lights-ref.rst

```diff
@@ -5,203 +5,514 @@ Many of the components of the FRC Control System have indicator lights that can
 
 Robot Signal Light (RSL)
 ------------------------
-
-.. image:: images/status-lights/rslLight.png
-
-- Solid ON - Robot On and Disabled
-- Blinking - Robot On and Enabled
-- Off - Robot Off, roboRIO not powered or RSL not wired properly.
+..
+  .. image:: images/status-lights/rsl.svg
```

Comment:
This isn't being added into the documentation. Maybe the indent and redundant `..` is throwing it off?

Reply:
I'm trying to selectively remove images from this PR until the LaTeX PDF build works properly to diagnose the issue. If you can provide any insight into which image causes that build to fail, that would be great.